### DISCUSSION TITLE
Creating new discussions

### DISCUSSION THREAD

**Discussion by r12a** [w3c/jlreq-d#50] [Announcements]
We are unable to generate notifications via the normal digest system for new GitHub Discussions, or new comments on a Discussion.  Therefore, we recommend that people creating a new discussion write to the list that normally receives notifications with text such as the following:

> I have just created a new GitHub Discussion at xxxx about yyy.  Please subscribe yourself to this discussion so that you can be notified of new comments and replies.  You can subscribe to the discussion by clicking on the Subscribe button in the right-hand column.

**Comment by kidayasuo** [comment]
All right. I will pass the suggestion on to TF members. Actually, it might be useful to create a wiki page. will discuss at today's call.
